release_creation/bundle/bundle_config.py

```diff
@@ -6,18 +6,20 @@
 from pydantic.dataclasses import dataclass
 from semantic_version import Version
 
-_OUTPUT_ARCHIVE_FILE_BASE = "dbt-core-all-adapters-snapshot"
+from release_creation.bundle.bundle_os import BundleOS
+
+_OUTPUT_ARCHIVE_FILE_BASE = "dbt-core-all-adapters-bundle"
 _FILE_DIR = os.path.dirname(os.path.realpath(__file__))
 
 
-def _get_local_os() -> str:
+def _get_local_os() -> BundleOS:
```

Comment:
This looks a lot like a dictionary. Also, have you thought about making it a `from_platform()` method on `BundleOS`?